src/UnderlineNav2/styles.ts

```diff
@@ -92,15 +92,23 @@ export const getLeftArrowHiddenBtn = (theme?: Theme) => ({
   opacity: 0,
   top: 0,
   bottom: 0,
-  left: 0
+  left: 0,
+  /* This is intentionally set to prevent any visual focus ring from showing up
+   * when a hidden element gets focus just to trigger assistive technology to reread */
+  '&[aria-disabled="true"]:focus': {
+    outline: 'none'
+  }
 })
 
 export const getRightArrowHiddenBtn = (theme?: Theme) => ({
   ...getArrowBtnStyles(theme),
   opacity: 0,
   top: 0,
   bottom: 0,
-  right: 0
+  right: 0,
+  '&[aria-disabled="true"]:focus': {
+    outline: 'none'
+  }
```

Comment:
```suggestion
  right: 0
```

Reply:
Because I updated the entire styles of the arrow buttons, I cannot apply your suggestions here, sorry but I removed them in the updated styles. I'm resolving this conversation here and we can discuss my concern further in the above comment 🙂